### DISCUSSION TITLE
Is scraping other user's activities really necessary?

### DISCUSSION THREAD

**Discussion by seb-afk** [SauceLLC/sauce4strava#91] [Q&A]
First of all, I love the extension and your work! Great stuff. I signed up as a Patreon to give the pro features a try and so far so good - cool little tool. But then today I noticed that I can literally scrape other user's entire Strava history without their consent... Which begs the question... Why? Is this not a tad controversial?

**Comment by mayfield** [comment]
Hi @sebastianbertoli.  Sauce can't see anything private.  It's just looking at public data or if the other athlete's profile is set to "Followers" and they allowed you to follow them.  It literally just looks at the profile page for a list of activities, the same way a human does, and the data streams are the same ones used for activity pages.  Consent is actually in Strava's hands here..
![image](https://user-images.githubusercontent.com/139316/156633028-73e2b518-f563-44f4-ba2c-de7962680be9.png)

As for the "why," I think George Mallory said it best, "Because it's there"

**Reply by seb-afk** [reply]
Hey @mayfield, thanks for the quick reply! I get that, it is one thing looking at the data, it is another one systematically scraping, processing and storing it... I am actually surprised Strava even allows this sort of stuff... 

> As for the "why," I think George Mallory said it best, "Because it's there"

Ok.
